sass/layout/_legacy-sidebar-left.scss

```diff
@@ -175,12 +175,15 @@
         justify-content: center;
         align-items: center;
         position: absolute;
-        margin-left: 50%;
+        margin-left: calc(50%);
```

Comment:
What's the reason for having `calc(50%)` here instead of just `50%`? I'm not that in depth with CSS, so I'm not sure if there's something substantially different between the two.

Reply:
Earlier Matt asked for a little cheat in horizontal margin so I used calc but now it's not necessary. I noticed it later. I read on https://docs.mattermost.com/developer/localization.html that translations are updated from mattermost Translation server so change of the string should be probably there. I don't have access to it. I let know about it in the first comment.